README.md

```diff
@@ -387,6 +391,79 @@ $ make flash
 [here](https://github.com/96boards/documentation/wiki/HiKeyUEFI#flash-binaries-to-emmc-))
 
 The board is ready to be booted.
+#### 5.5.2 Debian based / 96boards RPB (experimental)
+Start by getting the source and toolchain (see above), then continue by
+downloading the system image (root fs). There are two versions, the one that
+works best has a click through license which means that you have to manually
```

Comment:
The "other" (developer edition) doesn't seem to have the update-initramfs tools, the sources.list aren't up to date, so it's not just to "apt-get update" and "apt-get install update-initramfs" to get it working. As a result of that grub, initrd etc are updated/created when installing the kernel we've built on our own.

Where to find the rootfs image? The rule "make system-img" will tell that. In case of developer edition it will just be downloaded, in the "alip" version it will spawn a browser (using xdg-open) and point you directly to the image. See https://github.com/jbech-linaro/build/blob/debian_fs/hikey_debian.mk#L263-L272

Do you want me to clarify in the instructions that this will happen when running the make system-img command?